package/MDAnalysis/core/topologyattrs.py

```diff
@@ -338,6 +357,14 @@ def __init__(cls, name, bases, classdict):
                            "by subclassing core.selection.Selection")
                     warnings.warn(msg)
 
+        # TODO: remove in 3.0
+        if attrname == "tempfactors":
```

Comment:
So I might be completely forgetting what this meta class does again, but my read is that we warn about the deprecation if folks create a "tempfactor" attribute right? Is this the type of behaviour we want to to have? (i.e. bfactor is getting aliased to tempfactor, but do we want to warn users for using tempfactor?).

Reply:
It doesn't warn for users using or creating tempfactors -- the warning is only raised for `selcls` here which is the `bfactor` selection class.